.github/workflows/test.yml

```diff
@@ -71,3 +71,99 @@ jobs:
             Meadowlark-js/integration-tests/junit.xml
             Meadowlark-js/system-tests/junit.xml
           retention-days: 1
+
+  end-to-end:
+    name: End to End tests for ${{ matrix.config.db }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          [
+            { db: "Mongo", plugin: "@edfi/meadowlark-mongodb-backend" },
+            { db: "PGSQL", plugin: "@edfi/meadowlark-postgresql-backend" },
+          ]
+
+    defaults:
+      run:
+        working-directory: Meadowlark-js
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
+
+      - name: NPM caching
+        uses: actions/setup-node@17f8bd926464a1afa4c6a11669539e9c1ba77048 # v3.2.0
+        with:
+          node-version: "16"
+          cache: "yarn"
+          cache-dependency-path: "**/yarn.lock"
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Start OpenSearch Container
+        run: docker compose -f ./backends/meadowlark-opensearch-backend/docker/docker-compose.yml up -d
+
+      - if: ${{ matrix.config.db == 'PGSQL' }}
+        name: Start Postgres Container
+        run: docker compose -f ./backends/meadowlark-postgresql-backend/docker/docker-compose.yml up -d
+
+      - if: ${{ matrix.config.db == 'Mongo' }}
+        name: Start MongoDB
```

Comment:
Using MongoDB Action instead of setting up manual decreased the execution time from [7 minutes](https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/actions/runs/3349414681/jobs/5549393468) to [5 minutes](https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/actions/runs/3349518822/jobs/5549596165).

After seeing the time to setup Mongo, I don't think it's necessary to continue the effort to use in memory db